libraries/AP_Button/AP_Button.cpp

```diff
@@ -69,6 +69,24 @@ const AP_Param::GroupInfo AP_Button::var_info[] = {
     // @Range: 0 3600
     AP_GROUPINFO("REPORT_SEND", 5, AP_Button, report_send_time, 10),
 
+    // @Param: PIN1_PWM_MIN
+    // @DisplayName: First button Pin Minimum valid PWM input
+    // @Description: Minimum PWM input for switch to be pressed (zero for simple debounced switch)
+    // @User: Standard
+    AP_GROUPINFO("PIN1_PWM_MIN",  6, AP_Button, pin_params[0].pwm_min, -1),
+
+    // @Param: PIN1_PWM_MAX
+    // @DisplayName: First button Pin Minimum valid PWM input
+    // @Description: Maximum PWM input for switch to be pressed (zero for simple debounced switch)
+    // @User: Standard
+    AP_GROUPINFO("PIN1_PWM_MAX",  7, AP_Button, pin_params[0].pwm_max, -1),
+
+    // @Param: PIN1_FUNC
+    // @DisplayName: RC Channel function bound to this button
```

Comment:
Here too it would be good for the Display name to start with "Button".  The reason is that this is the human readable description so the most important word should appear first.

Reply:
I've fixed these in the new code.  It's a shame the interfaces aren't giving context - "Button" should come from the fact these are in the button library....